cloud-reactor/src/main/java/com/sequenceiq/cloudbreak/cloud/handler/InstanceStateHandler.java

```diff
@@ -43,6 +46,11 @@ public void accept(Event<GetInstancesStateRequest> event) {
                     instanceStateQuery.getCloudVmInstanceStatuses(
                             request.getCloudCredential(), cloudContext, request.getInstances());
             result = new GetInstancesStateResult(request, instanceStatuses);
+        } catch (CloudOperationNotSupportedException e) {
```

Comment:
You catch a different exception than throw in CumulusYarnInstanceConnector.java. is it ok?

Reply:
Thanks for checking.

Yes, this is intentional.

The following indicates an error, not that "check" is not supported.  This should still fail the flow as previously, that why the exception thrown is changed from `CloudOperationNotSupportedException` to the more generic `CloudConnectorException`.

https://github.com/hortonworks/cloudbreak/blob/33c6d01d80ce4135424b4eb15e218d05e93dbf09/cloud-cumulus-yarn/src/main/java/com/sequenceiq/cloudbreak/cloud/cumulus/yarn/CumulusYarnInstanceConnector.java#L63-L64

Reply:
I see, thanks for clarification